app/scripts/browser.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var _ = require('underscore');
+
+// Utilities for cross-browser compatibility.
+var Browser = {};
+
+// Map of CSS properties to property names used by the DOM in this browser.
+var normalizedProperties = {
+  'transform': _.find(['transform', 'WebkitTransform'], function (property) {
+    return document.documentElement.style[property] !== undefined;
+  }) || 'transform'
+};
+
+// Convert CSS properties and values to equivalent properties and values for
+// this browser.
+Browser.normalizeStyles = function (styles) {
+  var normalized = {};
+  _.forEach(_.keys(styles), function (property) {
+    if (Object.prototype.hasOwnProperty.call(normalizedProperties, property)) {
```

Comment:
Excellent and thoughtful reasoning. `Object.prototype.hasOwnProperty` can stay, then. :)